src/test/java/productDependency/CreateDependentCategoryTest.java

```diff
@@ -0,0 +1,193 @@
+import productDependency.CreateDependentCategoryPage;
+
+
+
+public class CreateDependentCategoryTest {
+	
+
+	CreateDependentCategoryPage dependentProductPage;
+    WebDriver driver,driver2;
+
+    @BeforeClass(alwaysRun=true)
+    public void setup(){
+        driver= BaseClasses.setup( "User1");
+        PropertyValExtractors p=new PropertyValExtractors();
+        p.getPropertyFile("test", "configuration.properties");
+        String url=p.getVal("url1");
+        driver.get(url);
+        //WaitClass.waitForUrl(driver, url, 10);
+        dependentProductPage=CreateDependentCategoryPage.getPage(driver, CreateDependentCategoryPage.class);
+
+
+}
+    @Test(groups={"Product Dependency Feature"},description="Add Edit Product")
+    public void userSignin() throws IOException{
+
+    	/*dependentProductPage.enterLoginID();
+    	dependentProductPage.enterPassword();
+    	dependentProductPage.selectCompany1();
+    	dependentProductPage.clickLoginButton();*/
+    	
+    	dependentProductPage.clickProductsTab();
+    	dependentProductPage.clickAddCategoryButton();
+    	dependentProductPage.enterCategoryName();
+    	dependentProductPage.clickSaveChangesButton();
+    	dependentProductPage.verifyConfirmationMsg();
+    	
+    	//Billing Category Product 1
+    	
+    	dependentProductPage.selectCategoryName();
+    	dependentProductPage.clickAddProductButton();
+    	dependentProductPage.enterEnglishDescription();
+    	dependentProductPage.enterProductCode();
+    	dependentProductPage.selectCompany();
+    	dependentProductPage.selectPriceCurrency();
+    	dependentProductPage.enterPriceRate();
+    	dependentProductPage.clickAddPriceButton();
+    	dependentProductPage.clickDependenciesPlus();
+    	dependentProductPage.selectProductCategoryDependency();
+    	dependentProductPage.selectProductPeriod();
+    	dependentProductPage.enterMin();
+    	dependentProductPage.clickDependencyPlusIcon();
+    	dependentProductPage.clickSaveChangesButton();
+    	dependentProductPage.verifyConfirmationMsg();
+    	
+    	
+    	//Billing Category Product 2
+    	
+    	dependentProductPage.clickProductsTab();
+    	dependentProductPage.selectCategoryName();
+    	dependentProductPage.clickAddProductButton();
+    	dependentProductPage.enterEnglishDescription1();
+    	dependentProductPage.enterProductCode1();
+    	dependentProductPage.selectCompany();
+    	dependentProductPage.selectPriceCurrency();
+    	dependentProductPage.enterPriceRate1();
+    	dependentProductPage.clickAddPriceButton();
+    	dependentProductPage.clickDependenciesPlus();
+    	dependentProductPage.selectProductCategoryDependency();
+    	dependentProductPage.selectProductPeriod1();
+    	dependentProductPage.enterMin1();
+    	dependentProductPage.enterMax1();
+    	dependentProductPage.clickDependencyPlusIcon();
+    	dependentProductPage.clickSaveChangesButton();
+    	dependentProductPage.verifyConfirmationMsg();
+    	
+    	
+    	//Billing Category Product 3
+    	
```

Comment:
@mayankchhajer no need to put space here

Reply:
Done